orc8r/cloud/deploy/bare-metal-ansible/README.md

```diff
@@ -0,0 +1,46 @@
+Deploying Magma on bare metal
+
+The following file needs to be edited before deployment can start:
+* ansible_vars.yaml
+
+The values that need to be customized are the IP settings for your network and
+the passwords which need to be generated. Additionally, you need to host a
+docker repo and helm chart repo for orc8r. Further details can be found at
+https://magma.github.io/magma/docs/orc8r/deploy_build#build-and-publish-helm-charts
+
+The deployment assumes the host running Ansible and the target hosts are
+running Ubuntu 18.04 or 20.04.
+
+The following variables must be set before starting:
+* orc8r_image_repo - Docker repo where controller, magmalte, and nginx containers are located
+* orc8r_helm_repo - Helm chart repo where orc8r chart is located
+* orc8r_domain - DNS domain where Magma services are published
+* orc8r_nms_admin_email - Initial login user for NMS
```

Comment:
minor nit: I would prefer calling this - orc8r_nms_admin_email -> orc8r_nms_user instead